roottest/root/treeproxy/stlvec/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+ROOTTEST_ADD_TEST(selector
+                  MACRO createtlorentzvecproxy.C
+                  COPY_TO_BUILDDIR tlorentzvec.root analyze.h analyze.C cutting.h cutting.C
+                  FIXTURES_SETUP root-treeproxy-stlvec-selector-fixture)
+
+ROOTTEST_ADD_TEST(tlorentzvec
+                  MACRO runtlorentzvec.C
```

Comment:
Side note: In the Makefile version ` tlorentzvecProxy.h` was compiled as a separate step, it is now compiled when running `runtlorentzvec.C`.  This is fine as `ROOTTEST_ADD_TEST` filters out the `ACLiC` info message about compilation.